src/components/bread-crumb/GroupBreadCrumbs.tsx

```diff
@@ -31,7 +32,7 @@ export const GroupBreadCrumbs = () => {
           </BreadcrumbItem>
           {subGroups.map((group, i) => (
             <BreadcrumbItem key={i} isActive={subGroups.length - 1 === i}>
```

Comment:
I think we can use the `isLastGroup()` function here as well.
```suggestion
            <BreadcrumbItem key={i} isActive={isLastGroup(i)}>
```